lib/fizzy/execute.cpp

```diff
@@ -300,7 +300,9 @@ Instance instantiate(const Module& module, std::vector<ImportedFunction> importe
     // Run start function if present
     if (module.startfunc)
     {
-        if (execute(instance, *module.startfunc, {}).trapped)
+        const auto funcidx = *module.startfunc;
+        assert(funcidx <= module.funcsec.size());
```

Comment:
This is an assert as it is ensured in the parser. However probably we'll need to make this a proper error, depending on the design of parser vs. instantiate.